R/is_prefixed.R

```diff
@@ -0,0 +1,22 @@
+#' Check if a factor is prefixed
+#' @param x a factor
+#' @export
+is_prefixed <- function(x) {
+  if (!is.factor(x))
+    stop("is_prefixed should be used only with a factor.")
+  l <- .get_prefixes.factor(x)
+  all(!is.na(l$code)) & all(!is.na(l$code))
```

Comment:
```suggestion
  all(!is.na(l$code)) && all(!is.na(l$code))
```
If both are scalar, it could save a nanosecond ;-)